internals/main.go

```diff
@@ -158,7 +158,18 @@ func (c *Client) do(
 	}
 	// create request
 	client := &http.Client{}
-	r, err := http.NewRequest("POST", c.proxyURL+parsedURL.Path, bytes.NewBuffer(data))
+  
+	// !NOTE: GET Requests are also converted to POST requests
+	// fmt.Println("[Interceptor] c.proxyURL+parsedURL.Path: ", c.proxyURL+parsedURL.Path+"?"+parsedURL.RawQuery) // For debugging purposes
+	
+  var url string
```

Comment:
Could you run `go fmt ./...` to format the code?

```suggestion
    var url string
```